silk/datadog_checks/silk/events.py

```diff
@@ -0,0 +1,28 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+ALERT_TYPES = {"INFO": "info", "ERROR": "error", "WARNING": "warning", "CRITICAL": "error"}
+
+
+class SilkEvent(object):
+    def __init__(self, raw_event, tags):
+        self.raw_event = raw_event
+
+        if self.raw_event:
+            self.payload = {
+                "timestamp": self.raw_event.get("timestamp"),
+                "event_type": self.raw_event.get("labels"),
```

Comment:
Does `event_type` in the event payload do anything? This parameter isn't documented in the API [docs](https://docs.datadoghq.com/api/latest/events/#post-an-event), but I'm seeing it documented in the Custom Agent Check [docs](https://docs.datadoghq.com/events/guides/agent/#submission). When I post an event to the API with `event_type: "EVENT, ACTION"`, it doesn't get saved as an attribute. Retrieving the event doesn't return `event_type`. Maybe, instead the event type labels can be appended to the event message or event title. Or maybe even as tags?

Reply:
In the Events Explorer page, `event_type` is used as a facet. I think it would still be useful to keep the `event_type` for filtering through different events.